hrldas/docs/README.ERA5

```diff
@@ -238,6 +232,14 @@ See the full documentation for a complete description of items, but these are th
 
  ZLVL = forcing height 
 
+For ERA5-Land data, will also need to change the four layer depth, as they are different in ERA5-Land data:
```

Comment:
I think I fixed this bug in the latest HRLDAS (https://github.com/NCAR/hrldas/commit/ba09bf052c22f1e0fcbb93065998f3903ce2c033). Users can still use the original Noah-MP soil thickness and the ERA-5 thickness will be interpolated into Noah-MP values. Could you please do a quick test of this?

Reply:
Thanks Cenlin, I did test the model can run with the original soil thickness (0.1, 0.3, 0.6 and 1.0). Will modify this part in the documentation.